homeassistant/components/sensor/tellduslive.py

```diff
@@ -85,7 +85,9 @@ def name(self):
     @property
     def state(self):
         """Return the state of the sensor."""
-        if self._type == SENSOR_TYPE_TEMP:
+        if not self.available:
+            return STATE_UNKNOWN
```

Comment:
Thanks all for comments. Changed to returning None instead.

Regarding not calling state at all if device is not available, I guess something like this is the change you are proposing?:

https://github.com/home-assistant/home-assistant/compare/dev...molobrakos:state?diff=split&name=state